code/modules/research/designs/anomaly.dm

```diff
@@ -0,0 +1,29 @@
+/datum/design/scienceglasses
+	name = "Science Goggles"
+	desc = "You expect those glasses to protect you from science-related hazards. Maybe you shouldn't."
+	id = "scienceglasses"
+	req_tech = list(Tc_MATERIALS = 1, Tc_ANOMALY = 2)
+	build_type = PROTOLATHE
+	materials = list(MAT_IRON = 700, MAT_GLASS = 2000)
+	category = "Anomaly"
+	build_path = /obj/item/clothing/glasses/science
+
+/datum/design/depth_scanner
+	name = "Depth Analysis Scanner"
+	desc = "Used to check spatial depth and density of rock outcroppings."
+	id = "depth_scanner"
+	req_tech = list(Tc_MATERIALS = 1, Tc_ANOMALY = 2)
+	build_type = PROTOLATHE
+	materials = list(MAT_IRON = 500, MAT_GLASS = 1000, MAT_GOLD = 200, MAT_SILVER = 200)
+	category = "Anomaly"
+	build_path = /obj/item/device/depth_scanner
+
+/datum/design/ano_scanner
```

Comment:
@JMWTurner if you actually try to triangulate the artifacts like that you're gonna waste your time, it takes so goddamn long that by the time you find a single one the round ended.

Here's PJB's guide to xenoarch:
1. Scan tile
2. If the scanner finds something, use hand pick before mining the tile with a regular pick.
3. If rocky debris drops, scan it, if the scanner goes crazy use your hand pick a couple times on the debris, and then your brush, tada artifact. If it doesn't go crazy move along RNG hated you.
4. Repeat on every asteroid tile you see.